app/src/main/java/com/hamdan/forzenbook/viewmodels/LoginViewModel.kt

```diff
@@ -150,17 +173,21 @@ class LoginViewModel @Inject constructor(
                         0
                     }
                     if (code in 200..299) {
```

Comment:
No 😁

Reply:
thats not in scope of this ticket, thats for FA-80, I have addressed this in that branch.

Reply:
You cannot just say no and then mark it as resolved without taking action. You don't even have a TODO comment here.

Reply:
Added a Todo and mentioned the ticket for it.